src/EventTickets/Repositories/EventRepository.php

```diff
@@ -37,6 +37,15 @@ public function getById(int $id): ?Event
             ->get();
     }
 
+    /**
+     * @unreleased
+     */
+    public function queryById(int $id): ModelQueryBuilder
```

Comment:
Are we using this `queryById()` method anywhere? Also, how is this intended to be used in addition to `getById()`?

Reply:
I noticed that the `queryById` methods are being called by methods in the `EventTickets` and `EventTicketTypes` models but have not been implemented. This pattern seems to be common in our repositories, so I implemented it accordingly.